verticapy/machine_learning/memmodel/cluster.py

```diff
@@ -234,14 +237,108 @@ def transform_sql(self, X: ArrayLike) -> list[str]:
 
 class KMeans(Clustering):
     """
-    InMemoryModel implementation of KMeans.
+    :py:mod:`verticapy.machine_learning.memmodel.base.InMemoryModel` implementation of KMeans.
 
     Parameters
     ----------
     clusters: ArrayLike
         List of the model's cluster centers.
     p: int, optional
         The p corresponding to one of the p-distances.
+        
+    .. note:: :py:mod:`verticapy.machine_learning.memmodel` are defined entirely by their attributes. For example, 'cluster centroids' and 'p value' define a KMeans model.
+    
+    Examples
+    --------
+
+    **Initalization**
+
+    Import the required module.
+
+    .. ipython:: python
+            :suppress:
+
+        from verticapy.machine_learning.memmodel.cluster import KMeans
+
+    A KMeans model is defined by its cluster centroids and the p value. In this example, we will use the following:
+
+    .. ipython:: python
+            :suppress:
+
+        clusters = [[0.5, 0.6], [1, 2], [100, 200]]
+        p=2        
```

Comment:
follow always the same formatting logic: put a space in between the '=' sign.
p = 2

Reply:
done